ocs_ci/helpers/vdbench_helpers.py

```diff
@@ -0,0 +1,622 @@
+                {"id": 1, "wd_id": 1, "elapsed": 120, "interval": 5, "iorate": "max"}
+            ],
+        },
+        "stress": {
+            "storage_definitions": [
+                {"id": 1, "lun": "/vdbench-data/stress1", "size": "5g", "threads": 4},
+                {"id": 2, "lun": "/vdbench-data/stress2", "size": "5g", "threads": 4},
+            ],
+            "workload_definitions": [
+                {"id": 1, "sd_id": 1, "rdpct": 30, "seekpct": 100, "xfersize": "4k"},
+                {"id": 2, "sd_id": 2, "rdpct": 70, "seekpct": 100, "xfersize": "64k"},
+            ],
+            "run_definitions": [
+                {"id": 1, "wd_id": 1, "elapsed": 1800, "interval": 30, "iorate": "max"},
+                {"id": 2, "wd_id": 2, "elapsed": 1800, "interval": 30, "iorate": "max"},
+            ],
+        },
+        "filesystem": {
+            "storage_definitions": [
+                {
+                    "id": 1,
+                    "fsd": True,
+                    "anchor": "/vdbench-data/fs-test",
+                    "depth": 2,
+                    "width": 4,
+                    "files": 10,
+                    "size": "1g",
+                    "format": "yes",
+                }
+            ],
+            "workload_definitions": [
+                {
+                    "id": 1,
+                    "sd_id": 1,
+                    "rdpct": 50,
+                    "xfersize": "8k",
+                    "threads": 2,
+                }
+            ],
+            "run_definitions": [
+                {
+                    "id": 1,
+                    "wd_id": 1,
+                    "elapsed": 120,
+                    "interval": 5,
+                    "iorate": "max",
+                }
+            ],
+        },
+    }
+
+    return configs
+
+
+def get_vdbench_pods(namespace, deployment_name):
+    """
+    Get list of Vdbench pod names for a deployment.
+
+    Args:
+        namespace: Kubernetes namespace
+        deployment_name: Name of the deployment
+
+    Returns:
+        list: List of pod names
+    """
+    try:
+        cmd = f"oc get pods -n {namespace} -l app={deployment_name} -o name"
+        result = run_cmd(cmd)
+        pods = [pod.replace("pod/", "") for pod in result.strip().split("\n") if pod]
+        return pods
+    except Exception as e:
+        log.error(f"Failed to get Vdbench pods: {e}")
+        return []
+
+
+def get_vdbench_logs(namespace, deployment_name, container="vdbench-container"):
+    """
+    Get logs from all Vdbench pods in a deployment.
+
+    Args:
+        namespace: Kubernetes namespace
+        deployment_name: Name of the deployment
+        container: Container name to get logs from
+
+    Returns:
+        dict: Dictionary mapping pod name to logs
+    """
+    pods = get_vdbench_pods(namespace, deployment_name)
+    logs_dict = {}
+
+    for pod in pods:
+        try:
+            cmd = f"oc logs -n {namespace} {pod} -c {container}"
```

Comment:
wouldn't it be quite heavy on the executor's memory? How about saving the logs into files?